src/main/java/Event.java

```diff
@@ -0,0 +1,17 @@
+public class Event extends Task{
```

Comment:
After the word "Task", there should be a single space before the "{" according to coding practice